### PR TITLE
Drawer component

### PR DESCRIPTION
**Pull request by StepankaBarotova**
Component interface: https://perzoinc.atlassian.net/wiki/spaces/DevX/pages/2227241175/Drawer

Figma design for the original MS component: https://www.figma.com/file/tdHdGPWz9wzdTH4MBHnmGN/MS-Components?node-id=2617%3A6351

Storybook:
Screenshot 2021-09-21 at 11 42 17

### CODE REVIEW

File: src/components/drawer/Drawer.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  className,
+  children,
+  ...rest
+}: DrawerContentProps) => (
+  <div className={classNames(buildClass(classStr), className)} {...rest}>
+    {children}
+  </div>
+);
+
+export const DrawerTitle: React.FC<DrawerContentProps> = ({
```
[the remark this answers is not in the document]
Comment (StepankaBarotova) [reply src/components/drawer/Drawer.tsx]: Is there a way to suppress this warning? I am not able to get rid of the duplicates.

File: src/components/drawer/Drawer.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      <div
+        {...rest}
+        role="dialog"
+        className={classNames(
+          prefix,
+          positionClasses,
+          relativeToClasses,
+          className
+        )}
+        style={{ width: width > minWidth ? width + 'px' : minWidth + 'px' }}
```
Comment (symphony-annaLlorens) [inline src/components/drawer/Drawer.tsx:92]: Is it possible to use rem instead of px?

**Review by symphony-annaLlorens — APPROVED** [review]
LGTM 👍

File: src/components/drawer/Drawer.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import Icon from '../icon';
+
+type DrawerProps = {
```
Comment (axeleriksson147) [inline src/components/drawer/Drawer.tsx:5]: Can we accept React.HTMLProps for this component?

File: src/components/index.ts [hunk 4]
```diff
@@ -18,6 +18,12 @@ import {
   ModalHeader,
   ModalTitle,
 } from './modal';
+import {
```
Comment (symphony-adnane) [inline src/components/index.ts:21]: can we do it better 
export * from ./drawer? (see line 35 for ex)

**Review by symphony-adnane — APPROVED** [review]

**Pull request merged**